### ISSUE TITLE
Replacing ResNet-101 with RestNet-152 pretrained models available in mxnet gives DeepLab FCN_Loss same values after several epochs?

### ISSUE THREAD

**Issue by chowkamlee81** [msracver/Deformable-ConvNets#93]
Iam doing simulation for semantic segmentation using basic version of deeplab.
 
I want to enhance the results by replacing resnet-101 model with resnet-152 with remaining   no change in other parameters. 
But the training **FCN-loss remains constant** even after several epochs/iterations.

Kindly suggest what could be the reason

**Comment by chowkamlee81** [comment]
@Oh233
@StOnEGiggity
@Franciszzj
@terrychenism
@terrychenism
@YuwenXiong
@ancientmooner
Kindly help for the above issue please
